addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -261,6 +261,7 @@ $input-placeholder-color: mix($input-bg, $input-color) !default;
 $input-disabled-bg: mix($input-color, $input-bg, 10%) !default;
 
 $form-check-input-border: ($input-border-width or 1px) solid $input-border-color !default;
+$form-check-input-border-radius: if($input-border-radius != null, if($input-border-radius < .1rem, 0, if($input-border-radius < .25rem, .15rem, .25rem)), null) !default;
```

Comment:
Nice, this computation ensures that the value inherited from `$input-border-radius` is not high enough to make a checkbox appear like a radio, good 👍 

I find the `.25rem` value a bit too conservative in case you want "go hard" on roundness, but increasing the limit would be too risky anyway.

Little nit-picking, add a comment saying that we're enforcing a maxim value of `.25rem` (BS default) to avoid the "checkbox = radio" scenario.

Thanks!